content/soarian/dstu2/diagnostic/diagnostic-report.md

```diff
@@ -18,15 +18,7 @@ The API returns the test names and values in reverse chronological order by resu
 <%= terminology_table(:soarian_diagnostic_report, :dstu2) %>
 
 ## Custom Extensions
-The content of these extensions must be visible to the user, and recorded with any storage or transmit activity along with the query/response date/time, as it represents the best representation of what clinicians recorded for the patient at the time of query.  
-
-| Name | Condition | Description |
-| --- | --- | --- |
-|[Narrative](https://www.hl7.org/fhir/dstu2/narrative.html)|No qualifying entries exist|**No relevant clinical data exists**|
-|[Narrative](https://www.hl7.org/fhir/dstu2/narrative.html)|Qualifying entries exist|The following labels and values from the list will be returned in HTML, if valued: **Test/Procedure Name, Test Value, Value Unit, Reference Range, micro (culture) results**|
-|[Narrative](https://www.hl7.org/fhir/dstu2/narrative.html)|Qualifying entries exist|Informational message that must be shown to the user: **Information provided is from YYYY-MM-DD through YYYY-MM-DD inclusive.**|
-
-**CAUTION:** The detail that is in the narrative may not be fully expressed in the structured portion of the FHIR response: The [narrative status](https://www.hl7.org/fhir/dstu2/narrative-definitions.html#Narrative.status) entry in the narrative is valued as "Additional" indicating that more information is provided in the narrative than is provided in the structure. Therefore, if the Application is using the structured entries for any purpose, care must be taken to ensure the clinical content/context of the narrative is preserved.  
```

Comment:
I like the positive phrasing of the implementation note.  We don't know for a fact that the delta between the narrative and coded portions is limited to date range, however.  How about we change the note to read: "The Narrative includes additional information pertain to the results, and should be shown to the user to ensure completeness of clinical content and context."